node/tests/OpenTelemetry.test.ts

```diff
@@ -242,6 +265,53 @@ describe("OpenTelemetry GlideClusterClient", () => {
         TIMEOUT,
     );
 
+    it.each([ProtocolVersion.RESP2, ProtocolVersion.RESP3])(
+        `GlideClusterClient test percentage requests config_%p`,
+        async (protocol) => {
+            const client = await GlideClusterClient.createClient({
+                ...getClientConfigurationOption(
+                    cluster.getAddresses(),
+                    protocol,
+                ),
+            });
+            OpenTelemetry.setSamplePercentage(0);
+            // wait for the spans to be flushed and removed the file
+            await new Promise((resolve) => setTimeout(resolve, 500));
+
+            if (fs.existsSync(VALID_ENDPOINT_TRACES)) {
+                fs.unlinkSync(VALID_ENDPOINT_TRACES);
+            }
+
+            await client.set("test_key", "value");
+            await new Promise((resolve) => setTimeout(resolve, 500));
+            // check that the spans not exporter to the file due to the requests percentage is 0
+            expect(fs.existsSync(VALID_ENDPOINT_TRACES)).toBe(false);
+
+            OpenTelemetry.setSamplePercentage(100);
+
+            // Execute a series of commands sequentially
+            for (let i = 0; i < 50; i++) {
+                const key = `test_key_${i}`;
+                await client.set(key, `value_${i}`);
+                await client.get(key);
+            }
+
+            // Wait for spans to be flushed to file
+            await new Promise((resolve) => setTimeout(resolve, 5000));
+
+            // Read the span file and check span name
+            const { spanNames } = readAndParseSpanFile(VALID_ENDPOINT_TRACES);
+
+            expect(spanNames).toContain("Set");
+            expect(spanNames).toContain("Get");
+
+            if (fs.existsSync(VALID_ENDPOINT_TRACES)) {
+                fs.unlinkSync(VALID_ENDPOINT_TRACES);
+            }
```

Comment:
can you move it to a dedicated teardown function - e.g. teardown_otel_test, that will cleanup all otel files